package.json

```diff
@@ -48,6 +48,11 @@
                 "when": "editorTextFocus"
             },
             {
+                "key": "ctrl+[",
+                "command": "extension.vim_esc",
```

Comment:
I would prefer to send the actual ctrl+[ key sequence here, so that you can test it later on. (It's also important for remapping!)